ui/widgets/button.js

```diff
@@ -242,6 +248,10 @@ $.widget( "ui.button", {
 				this._updateTooltip();
 		}
 
+		if ( this.options.textDir && ( key === "label" || key === "textDir" ) ) {
```

Comment:
What about handling when `textDir` changes? This seems like a place where we probably need to leverage `_setOptions()` instead of `_setOption()`.